### PR TITLE
Add Eslint and Travis-CI to the project

### LINKED ISSUE #18: Add eslint and travis-ci
Many of our pull request have had the same comment of adding newlines to the end of files. We should add eslint into the project, and add travis-ci to ensure pull requests follow the code style rules.

### PR DESCRIPTION
**Pull request by Zeragamba**
[fixes #18]

Will require @camerongillette to enable travis-ci for this project.

See my travis build: https://travis-ci.org/SpyMaster356/factorio_blueprint_editor

### CODE REVIEW

**Review by palexcast — APPROVED** [review]
LGTM

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-console": [ "off" ],
+        "indent": [ "error", 4 ],
+        "linebreak-style": [ "error", "windows" ],
```
Comment (Zeragamba) [inline .eslintrc.js:9]: Looks like the original version was developed on windows. Should we update the line endings to be unix style?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/camerongillette/factorio_blueprint_editor.git"
+  },
+  "keywords": [
+    "Factorio",
+    "Blueprints",
+    "Tool"
+  ],
+  "author": "camerongillette, Elxeno",
```
Comment (Zeragamba) [inline package.json]: I think this is the right author. should we change it?

Comment (ryepup) [reply package.json]: It probably doesn't matter, but I think npm wants something a little different here, see https://docs.npmjs.com/files/package.json#people-fields-author-contributors

```js
 "author": { "name": "Elxeno" },
 "contributors": [
    {"name": "camerongillette"}
  ]
```

**Comment by Zeragamba** [comment]
Any update on this?

**Comment by ryepup** [comment]
@SpyMaster356 can you please resolve the merge conflict in js/script.js?

**Comment by Zeragamba** [comment]
Changes made, however my Travis build fails due to a wonky line that was added as part of #30. I have left a comment there.

**Review by ryepup — APPROVED** [review]

**Comment by Zeragamba** [comment]
I'm going to merge this pull request in so that the `npm run lint` command is available to all contributors.

**Pull request merged**